packages/react-core/src/demos/Card/Card.md

```diff
@@ -652,17 +652,16 @@ class CardViewBasic extends React.Component {
               </Card>
               {filtered.map((product, key) => (
                 <Card
-                  hasSelectableInput
                   isCompact
-                  isSelectableRaised
+                  isClickable
+                  isSelectable
+                  isSelected={selectedItems.includes(product.id)}
                   key={product.name}
                   id={product.name.replace(/ /g, '-')}
                   onKeyDown={(e) => this.onKeyDown(e, product.id)}
                   onClick={() => this.onClick(product.id)}
```

Comment:
We should remove both the onKeyDown and onClick here and instead pass an onChange property to the `selectableActions` prop in CardHeader. onKeydown won't be needed with the onChange, and based on how we're suggesting a clickable and selectable card should behave we don't want to "select" the card when we click anywhere within it.

Reply:
Thanks Eric, this is a great idea.  I thought we would want the full card to be selectable as it already had been, but due to the presence of the kebab it makes sense for only the checkbox area to be selectable now.

I added these updates in the [Card View refactor PR](https://github.com/patternfly/patternfly-react/pull/9359/commits/758641fea1d3e486ed9ffd3e077af2a4fb9ef166) because passing `onChange` this way lends itself better to TS. and was causing some issues when used within a class component